src/io/flutter/actions/InstallSdkAction.java

```diff
@@ -31,15 +53,166 @@ String getLinkText() {
     }
   }
 
+  private static class GitCloneAction extends InstallAction {
+    private final FlutterGeneratorPeer myPeer;
+
+    public GitCloneAction(FlutterGeneratorPeer peer) {
+      myPeer = peer;
+    }
+
+    @Override
+    void perform() {
+      @SuppressWarnings("DialogTitleCapitalization") final FileChooserDescriptor descriptor =
```

Comment:
I like this idea especially if we're reinforcing something that we suggest in the install docs.